### PR TITLE
Add shell instruction to serving Dockerfiles

### PR DESCRIPTION
**Pull request by RobertLucian**
Coming as an improvement to https://github.com/cortexlabs/cortex/issues/1086#issuecomment-635913877. Closes https://github.com/cortexlabs/cortex/issues/1086.

Instead of having to run `RUN /bin/bash -c "pip install ..."` for a custom image based on any of Cortex's base serving images or inside `dependencies.sh`, the user now only has to add `RUN pip install ...`.

Also, clarify the docs about pip/conda installations for extended Dockerfiles.

---

checklist:

- [x] run `make test` and `make lint`
- [x] manually test Python/TensorFlow/ONNX deployments.
- [x] update docs and add any new files to `summary.md` (view in [gitbook](https://docs.cortex.dev/v/master) after merging)

### CODE REVIEW

File: docs/deployments/system-packages.md [hunk 1]
```diff
@@ -68,6 +68,9 @@ FROM cortexlabs/python-predictor-cpu-slim:master
 RUN apt-get update \
     && apt-get install -y tree \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN pip install pandas \
```
Comment (deliahu) [inline docs/deployments/system-packages.md:72]: Do you mind adding `--no-cache-dir` to `pip install`? Also do you know if there is an equivalent for conda?

File: images/onnx-predictor-cpu/Dockerfile [hunk 2]
```diff
@@ -33,14 +33,10 @@ RUN /opt/conda/bin/conda create -n env -c conda-forge python=$PYTHONVERSION pip
...
+SHELL ["/bin/bash", "-c"]
 
 COPY pkg/workloads/cortex/serve/requirements.txt /src/cortex/serve/requirements.txt
-
-# use /bin/bash to run in conda environment
-RUN /bin/bash -c \
-    "pip install --no-cache-dir \
-        -r /src/cortex/serve/requirements.txt \
-        onnxruntime==1.2.0"
+RUN pip install --no-cache-dir -r /src/cortex/serve/requirements.txt onnxruntime==1.2.0
```
Comment (deliahu) [inline images/onnx-predictor-cpu/Dockerfile:39]: do you mind putting `-r /src/cortex/serve/requirements.txt` and `onnxruntime==1.2.0` on separate new lines? I find it a little easier to see what's installed if it's one item per line. (same applies for the other Dockerfiles if you agree)

File: docs/deployments/system-packages.md (hunk 1 above)
Comment (RobertLucian) [reply docs/deployments/system-packages.md]: Yep. As for `conda`, I've added `conda clean -a` afterward - that appears to be the only equivalent.

File: images/onnx-predictor-cpu/Dockerfile (hunk 2 above)
Comment (RobertLucian) [reply images/onnx-predictor-cpu/Dockerfile]: Of course!

**Comment by deliahu** [comment]
Looks great, will merge now, thanks!

**Pull request merged**